apimodels/agent_models.go

```diff
@@ -129,10 +131,18 @@ func (ch *CreateHost) Validate() error {
 		catcher.Add(errors.New("aws_access_key_id, aws_secret_access_key, key_name must all be set or unset"))
 	}
 
-	if ch.NumHosts > 10 || ch.NumHosts < 0 {
-		catcher.Add(errors.New("num_hosts must be between 1 and 10"))
-	} else if ch.NumHosts == 0 {
-		ch.NumHosts = 1
+	if ch.NumHosts == "" {
+		ch.NumHosts = "1"
```

Comment:
this seems like it should be an error. It was before.

Reply:
That error check is preserved on 141.